install-dev/fixtures/fashion/install.php

```diff
@@ -42,4 +42,23 @@ public function createEntityCustomer($identifier, array $data, array $data_lang)
 
         return $this->createEntity('customer', $identifier, 'Customer', $data, $data_lang);
     }
+
+    /**
+     * @{inheritdoc}
+     */
+    public function populateFromXmlFiles()
+    {
+        parent::populateFromXmlFiles();
+
+        /**
+         * Refresh facetedsearch cache
+         */
+        /** @var Ps_Facetedsearch */
+        $facetedSearch = Module::getInstanceByName('ps_facetedsearch');
+        if ($facetedSearch) {
+            $facetedSearch->buildLayeredCategories();
+            $facetedSearch->rebuildLayeredCache();
+            $facetedSearch->fullPricesIndexProcess(0, false);
```

Comment:
I'm not in favor having a strong dependency between the core and a specific module. If one day one of these method disparears without taking care of the compatibility, all installation may crash.

May I suggest a hook instead?

Reply:
No problem, I'll try something else :)

Reply:
I tried with `to_reset` in https://github.com/PrestaShop/PrestaShop/blob/develop/themes/classic/config/theme.yml#L54 but because there's no data, nothing happened.
And there's no hook in the whole installer. And I think fixtures must be the last step of the installer :( I'm stuck :/

Reply:
Should be better using the module manager :)